<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalButtonBar</class>
 <widget class="QWidget" name="GlobalButtonBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#content_widget{
background:#1d2129;
border-radius:30px;
}


#btn_camera{
border:none;
background:transparent;
background-image:url(:img/meeting_big_cam);
background-position:center;
background-repeat:no-repeat;
}

#btn_hang{
border:none;
background:transparent;
background-image:url(:img/hang_up);
background-position:center;
background-repeat:no-repeat;
}


#btn_mic{
border:none;
background:transparent;
background-image:url(:img/meeting_big_mic);
background-position:center;
background-repeat:no-repeat;
}


#btn_record{
border:none;
background:transparent;
background-image:url(:img/meeting_record);
background-position:center;
background-repeat:no-repeat;
}


#btn_setting{
border:none;
background:transparent;
background-image:url(:img/meeting_big_setting);
background-position:center;
background-repeat:no-repeat;
}


#btn_share{
border:none;
background:transparent;
background-image:url(:img/meeting_share);
background-position:center;
background-repeat:no-repeat;
}


#btn_user_list{
border:none;
background:transparent;
background-image:url(:img/meeting_user_list);
background-position:center;
background-repeat:no-repeat;
}

#line{
background:#4E5969;
}



</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="content_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="PushButtonWarp" name="btn_mic">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButtonWarp" name="btn_camera">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButtonWarp" name="btn_share">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButtonWarp" name="btn_record">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButtonWarp" name="btn_user_list">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButtonWarp" name="btn_setting">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButtonWarp" name="btn_hang">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButtonWarp</class>
   <extends>QPushButton</extends>
   <header>push_button_warp.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
